### ISSUE TITLE
Change theme to match `flake.parts`

### ISSUE THREAD

**Issue by srid** [flake-parts/community.flake.parts#7]
@roberth Could you confirm that this is what you want community.flake.parts look like?

<img width="952" alt="image" src="https://github.com/flake-parts/community.flake.parts/assets/3998/b0f8fe1b-d6fe-4aa2-8fc6-01b4b5b8203e">

**Comment by srid** [comment]
And is this a builtin theme of mdbook? Where is it defined, btw?

**Comment by roberth** [comment]
Yes, it is a good example for a documentation theme, because it is a calm, flat style that doesn't distract from the content.
Also, a very similar style makes the two sites look more coherent.

It is mostly the default "light" theme, with a few overrides:
https://github.com/hercules-ci/flake.parts-website/blob/main/site/flake-parts.css

The rest of the theme is as defined upstream by mdBook [here](https://github.com/rust-lang/mdBook/tree/master/src/theme).

**Comment by roberth** [comment]
The Obsidian theme has a couple of flaws, but the general looks is ok.

- Saturated green text ("Released") on pastel green background: ugly and might be insufficient contrast (guessing; I'm no color vision expert)
- Note background are label background are also dissonant.
- Something's jarring about those colors in dark mode too. Maybe because the label background intensity matches the "Released" text color instead of its background.
- Drop shadows look dated
- Corner rounding is inconsistent
- Their link color looks like the old color for already-visited links, but I suppose we'd use the flake-parts logo color anyway. At least that's what I did on flake.parts. (but only in the menu? - might want to apply that more broadly, but maybe not as bright? needs testing)
